pkg/ctime/ctime_linux_64.go

```diff
@@ -1,14 +0,0 @@
-// +build !arm,!386,linux
```

Comment:
Should this still be "// +build linux"?

Reply:
If the file is gone, I don't think we need a tag for it

Reply:
@mheon, DOH! you're right.  Too many reviews today.